.circleci/config.yml

```diff
@@ -3,6 +3,7 @@ version: 2.1
 
 orbs:
   go: circleci/go@1.7.1
+  git-shallow-clone: guitarrapc/git-shallow-clone@2.4.0
```

Comment:
Any reason to add this btw? @matej-g

Reply:
The CI was failing because of https://cloud-native.slack.com/archives/CL25937SP/p1666777294865379, it's just a cherry-pick from the `main` but should not affect the release itself.